prototypes/prototype.adoc

```diff
@@ -0,0 +1,44 @@
+---
+authors: Han Solo <han@example.com>, Frodo Baggins <frodo@example.com>
+state: predraft
+---
```

Comment:
blank link before the title please... or perhaps let's move the title to after all the adoc attributes?